packages/terra-table/tests/jest/ColumnResizeHandle.test.jsx

```diff
@@ -105,7 +105,7 @@ describe('ColumnResizeHandle', () => {
   it('sets the appropriate prop values on space keydown', () => {
     const wrapper = mountWithIntl(
       <ColumnContext.Provider value={{ setColumnHeaderAriaLiveMessage: jest.fn() }}>
-        <ColumnResizeHandle />
+        <ColumnResizeHandle setIsActive={jest.fn()} />
```

Comment:
We are adding a new prop to pass a function, but it does not seem like we are testing to ensure it was called.  What are we attempting to validate?